server/src/test/java/org/elasticsearch/transport/RemoteConnectionManagerTests.java

```diff
@@ -89,7 +90,7 @@ public void testGetConnection() {
         assertEquals(1, versions.size());
     }
 
-    public void testResolveRemoteClusterAlias() {
+    public void testResolveRemoteClusterAlias() throws ExecutionException, InterruptedException {
```

Comment:
Change from this PR: https://github.com/elastic/elasticsearch/pull/92584 -- pending approval from distributed team.